bazel_codegen/pubspec.yaml

```diff
@@ -5,14 +5,17 @@ homepage: https://github.com/dart-lang/bazel
 version: 0.1.6
 
 environment:
-  sdk: ">=1.8.0 <2.0.0"
+  sdk: ">=1.25.0-dev.7.0 <2.0.0"
```

Comment:
nevermind - we'd have to cut a new rules_dart and dazel for anyone to pick this up and we can bump versions of the sdk there if we need to.

Would any of these changes be considered breaking?